bigbluebutton-client/branding/default/style/css/BBBDefault.css

```diff
@@ -44,10 +44,10 @@ ToolTip {
   cornerRadius: 5;
   borderStyle: solid;
   borderThickness: 1;
-  paddingBottom: 3;
-  paddingTop: 3;
-  paddingLeft: 3;
-  paddingRight: 3;
+  paddingBottom: 2;
+  paddingTop: 2;
+  paddingLeft: 2;
+  paddingRight: 2;

```

Comment:
This PR has nothing to do with the whiteboard, why do we change the toolbar style? If we use the same style for the chat toolbar, we should create a new style for it.